### PR TITLE
scicomp/git-annex: Add mentions of DataLad

### PR DESCRIPTION
**Pull request by rkdarst**
- Based on mastodon threads.
- Review: check accuracy, check if the RDM-focus is justified, check
  if it should be even more strongly recommended than is written here.

### CODE REVIEW

**Comment by rkdarst** [comment]
@jcolomb - do you think this is accurate?  Should I be even more bold and tell everyone to start with datalad as an interface, and get all the things you see below in a more usable package (yet still usable independently)?

**Comment by jcolomb** [comment]
may well depend on your readership, the strength of datalad compared to git annex directly is in (as far as I understand it):
- setting rules about what is annexed and what is not, then adding everything to git or to git annex via a single command.
- using submodules
- run command (get files,  do the analysis, drop them again + add metadata in git commit)

For most of my readership, datalad is still too complicated...

the GIN-cli is closer to git-annex, but works only for GIN repositories I think. (https://github.com/G-Node/gin-cli)

I would add a note at the beginning redirecting readers to these tools depending on their use-case.

BTW: for me it was super interesting, helped to understand the core functionalities, and will help me thinking about the tool. You may want to explain better what you meant with large files are put in chunks, I am not sure I understood.

File: scicomp/git-annex.rst [hunk 1]
```diff
@@ -5,6 +5,9 @@ Git-annex for data management
```
Comment (jcolomb) [inline scicomp/git-annex.rst:5]: This presentation is mostly meant for people with experience with git and the command line, who want to know more about git annex. If you feel overwhelmed by the content, you may want to check the [datalad handbook](http://handbook.datalad.org/en/latest/) for a smoother entry in the use of command line and git annex functionalities.
Also note that git annex solves the issue of big files, but not numerous files (git starts to slow down with several hundreds of files). To deal with numerous files, one can use git submodules, again datalad may be a tool of choice for implementing submodules.

Comment (jcolomb) [reply scicomp/git-annex.rst:5]:
```suggestion
We hope you will enjoy reading this presentation of the main functionalities of git annex. It proves interesting to follow, as to understand the logic of the tool, even if you may move to using datalad later on.
```

**Review by jcolomb — COMMENTED** [review]
putative intro, you may want to add a word about the quality

File: scicomp/git-annex.rst (hunk 1 above)
Comment (rkdarst) [reply scicomp/git-annex.rst:5]: I've added a lot of the above to the comment on the top (slightly rephrased but it was a good idea!)  I'm sorry for taking so long to review.

**Pull request merged**